### DISCUSSION TITLE
How to install updates?

### DISCUSSION THREAD

**Discussion by Dery36** [rsjaffe/MIDI2LR#564] [Q&A] [answered]
I"m so sorry to ask this question, but I don't see the answers anywhere. I've been using MIDI2LR with X-TOUCH for years now and love it. But I haven't installed any updates because I don't want to mess anything up. I've looked on youtube for a video on how to do it but can't find anything. Where are the instructions on how to install the updates? Thanks in advance.

**Comment by rsjaffe** [comment, marked as answer]
See the wiki: https://github.com/rsjaffe/MIDI2LR/wiki#updating-from-a-previous-version
